lib/shared-sku-app/src/SkuPicker/styles.ts

```diff
@@ -1,26 +1,39 @@
 import { css } from 'emotion';
 import tokens from '@contentful/forma-36-tokens';
 
-/**
- * Returns the target height of the modal window body in pixels.
- * Relative units were causing multiple resizings of the iframe.
- */
-function getBodyHeight() {
-  const HEADER_HEIGHT = 114;
+const HEADER_HEIGHT = 114;
+
+const SHOULD_HAVE_STICKY_HEADER = window.outerHeight >= 900;
```

Comment:
it is the iframe and not the webapp, it could not have been the webapp as that info is not available from within an iframe without `postMessage` and an sdk method.

Reply:
Cool, thx for the info!